akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -262,57 +263,174 @@ class CircuitBreaker(
    * @param callback Handler to be invoked on state change
    * @return CircuitBreaker for fluent usage
    */
-  def onOpen(callback: ⇒ Unit): CircuitBreaker = onOpen(new Runnable { def run = callback })
+  def onOpen(callback: ⇒ Unit): CircuitBreaker = addOnOpenListener(new Runnable { def run = callback })
 
   /**
-   * Java API for onOpen
+    * Java API for onOpen
```

Comment:
All of these comments gets re-indented by the builds source code formatter, making it look like there are changes in the file. Please indent them like this:

```
/**
 * ...
 */